iso15118/secc/failed_responses.py

```diff
@@ -475,14 +481,31 @@ def init_failed_responses_iso_v2() -> dict:
         AuthorizationReqV2: AuthorizationResV2(
             response_code=ResponseCodeV2.FAILED, evse_processing=EVSEProcessing.FINISHED
         ),
-        ChargeParameterDiscoveryReq:
-        # TODO: Need to find a way to circumvent the root_validator
-        ChargeParameterDiscoveryRes(
-            response_code=ResponseCodeV2.FAILED, evse_processing=EVSEProcessing.FINISHED
+        ChargeParameterDiscoveryReq: ChargeParameterDiscoveryRes(
+            response_code=ResponseCodeV2.FAILED,
+            evse_processing=EVSEProcessing.FINISHED,
+            ac_evse_charge_parameter=ACEVSEChargeParameter(
+                ac_evse_status=ACEVSEStatus(
+                    notification_max_delay=0,
+                    evse_notification=EVSENotification.NONE,
+                    rcd=False,
+                ),
+                evse_nominal_voltage=PVEVSENominalVoltage(
+                    multiplier=0, value=400, unit=UnitSymbol.VOLTAGE
+                ),
+                evse_max_current=PVEVSEMaxCurrent(
+                    multiplier=0, value=32, unit=UnitSymbol.AMPERE
```

Comment:
Do you want to set the value field to 0.
Just to avoid the case where the recipient ignores the response_code and still decide to use the response?

Reply:
Done